server/svix-server/src/error.rs

```diff
@@ -28,6 +28,8 @@ pub enum Error {
     Validation(String),
     /// Any kind of HttpError
     Http(HttpError),
+    /// Timeout consuming from queue
+    QueueTimeout(String),
```

Comment:
It should be more like Http errors, so:
```
Queue(QueueError)
```
And then the timeout would be one of their queue errors.

Though I think we don't need it anymore?